templates/macros/display-text.html.twig

```diff
@@ -126,10 +126,11 @@
     {% endif %}
 {% endmacro %}
 
-{% macro displayConsentPanelStatus(participant, field, showConsentPDFs = '') %}
+{% macro displayConsentPanelStatus(participant, field, showConsentPDFs = '', showCheck = false) %}
     {% set consentValues = ['SUBMITTED', 'SUBMITTED_NO_CONSENT', 'SUBMITTED_NOT_SURE', 'SUBMITTED_INVALID'] %}
     {% if attribute(participant, field) in consentValues %}
         {% set time = attribute(participant, field ~ 'Authored') %}
+        {% if showCheck %}<i class="fa fa-check text-success" aria-hidden="true"></i>{% endif %}
```

Comment:
I think the green tick needs to be displayed only when the value is `SUBMITTED`. Might need to check with Gwen and confirm.

Reply:
Setup a new function to match the workqueue logic for the consent.